eq-author/src/App/page/PropertiesPanel/GroupedAnswerProperties/AnswerProperties/index.js

```diff
@@ -29,6 +41,21 @@ export class UnwrappedAnswerProperties extends React.Component {
       id,
       properties,
     });
+
+    if (type === "Date" && propName === "format") {
```

Comment:
As you've imported `DATE` above, you should use it within this statement instead of `"Date"`

Reply:
Fixed